invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/user_dashboard/components/requests.js

```diff
@@ -128,7 +128,7 @@ export const RequestsResults = ({
 
 export function RequestsResultsGridItemTemplate({ result, index }) {
   return (
-    <Card fluid key={index} href={`/requests/${result.metadata.id}`}>
+    <Card fluid key={index} href={`/me/requests/${result.metadata.id}`}>
```

Comment:
Shouldn't this be obtained through the user-dashboard-community-submission variable defined in the config.py

Reply:
there is a separate issue created for api urls creation, which is not the scope of this PR